process/subprocesses/ghsci.py

```diff
@@ -844,18 +844,35 @@ def _network_data_setup(self, r):
             r['network']['polygon_iteration'] = False
         if 'connection_threshold' not in r['network']:
             r['network']['connection_threshold'] = None
+        
+        # Setup pedestrian intersections
         if (
-            'intersections' in r['network']
-            and r['network']['intersections'] is not None
+            'intersections_pedestrian' in r['network']

```

Comment:
As per elsewhere, I'm not sure the distinction between pedestrian and cycling for intersection evaluation in this way is necessary, and so may be an over-complication.  Perhaps there could be a different approach where the single set of configured intersections are post hoc evaluated (as suggested elsewhere) eg for 'junction stress' --- but this wouldn't strictly require a separate dataset, it would instead post hoc identify some intersections as more suitable than others.  That could be interesting for an interactive map perhaps, or to account for in routing.  Do you think something like that could work @ryn-trnr?